curricula/self_paced/part_1/z230-1c.md

```diff
@@ -0,0 +1,245 @@
+Python 0-30 for Scientists
+==========================
+
+Part 1c - Writing Function to Compute Wind Chill Index
+--------------------------------------
+
+This is intended to pick off right where part 1b left off- you 
+had just commited your new script that reads the file, saving 
+the variables of date, time, and tempout in a data dictionary.
+
+In this section you will compute wind chill index by writing
+your first function and learning about basic math operators.
+
+1. [python] Okay, now that you've read the data in a way that
+   is easy to modify later, it is time to actually do something 
+   with the data.
+   
+   Compute the *wind chill* factor, which is the cooleing 
+   effect of the wind. As wind speed increases the rate at 
+   which a body loses heat increases. The formula for this is:
+   
+   WCI = a + (b * Temperature) - (c * WindSpeed^2) + 
+      (d * Temperature * WindSpeed^2)
```

Comment:
I'm not sure how to render this better.  Not sure what the best approach is...  However, if markdown can't render equations nicely, then I think we might want to move this content over to Jupyter Notebooks.  They can display Latex equations.

Reply:
That's a good idea.

Reply:
This is done in the .ipynb files